docs/management/manage-locally/api.md

```diff
@@ -46,7 +50,54 @@ Example:
 pending_task_timeout = 172800
 ```
 
+After changing the configuration, restart the toolstack with `xe-toolstack-restart`. 
+
+#### Enable/Disable HSTS
+
+If you want to enable HTTP Strict Transport Security: 
+- Create a new configuration file in `/etc/xapi.conf.d/`.
+- You can name it `/etc/xapi.conf.d/hsts.conf`.
+- Define the new value for `hsts_max_age`, in seconds.
+
+Example:
+```ini
+# Set HSTS retained for 1y
+hsts_max_age = 31536000
+```
+:::tip
+Common values are:
+- 1 year --> 31536000
+- 2 years --> 63072000
+:::
+After changing the configuration, restart the toolstack with `xe-toolstack-restart`. 
+
+Example with one command to enable HSTS and restart XAPI (example sets 1 year = 31536000):
+```bash
+echo "hsts_max_age = 31536000" > /etc/xapi.conf.d/hsts.conf' && sudo xe-toolstack-restart
+```
+
+:::note
+Please wait (~30s) for the service to become fully available before issuing further XAPI requests.
+:::
```

Comment:
There are better ways to check that XAPI is fully started: checking the status of xapi-wait-init-complete, IIRC.